core/node/options_payments.go

```diff
@@ -28,8 +28,14 @@ type OptionsPayments struct {
 	BCTimeout                      time.Duration
 	HermesPromiseSettlingThreshold float64
 	SettlementTimeout              time.Duration
+	SettlementRecheckInterval      time.Duration
 	ConsumerDataLeewayMegabytes    uint64
 	ProviderInvoiceFrequency       time.Duration
 	MaxUnpaidInvoiceValue          *big.Int
 	HermesStatusRecheckInterval    time.Duration
+	BalanceFastPollInterval        time.Duration
+	BalanceFastPollTimeout         time.Duration
+	BalanceLongPollInterval        time.Duration
```

Comment:
Pattern suggests there should be `BalanceLongPollTimeout` :)

Reply:
the long poll never times out, it's continuous and inifite.